gobblin-modules/gobblin-kafka-09/src/main/java/org/apache/gobblin/kafka/writer/Kafka09DataWriter.java

```diff
@@ -96,6 +107,7 @@ public Kafka09DataWriter(Properties props) {
   public Kafka09DataWriter(Producer producer, Config config)
   {
     this.topic = config.getString(KafkaWriterConfigurationKeys.KAFKA_TOPIC);
+    provisionTopic(topic,config);
```

Comment:
provisioning of topic should be optional and opted-in via a configuration

Reply:
It is already there based on setting the zookeeper cluster property
```
 String zooKeeperPropKey = KafkaWriterConfigurationKeys.CLUSTER_ZOOKEEPER;
	    if(!config.hasPath(zooKeeperPropKey)){
	    	log.debug("Topic "+topicName+" is configured without the partition and replication");
	    	return;
	    }
```